src/ipfs.io/docs/index.html

```diff
@@ -12,8 +12,6 @@
 
 ## Documentation
 
-- [Install](docs/install)
-- [Getting Started](docs/getting-started)
```

Comment:
I think we should keep them for completeness and higher-likelihood that the user finds what they want.  Navigating websites is actually not straight forward. people miss obvious stuff all the time. tons of sites employ links that lead to the same place under different semantic locations (like navbar and this menu).